pkg/rancher-desktop/assets/scripts/install-k3s

```diff
@@ -41,3 +41,12 @@ fi
 ln -s -f "${K3S_DIR}/${K3S}" /usr/local/bin/k3s
 # The file system may be readonly (on macOS)
 chmod a+x "${K3S_DIR}/${K3S}" || true
+
+# Make sure any old manifests are removed before configuring k3s again
+MANIFESTS=/var/lib/rancher/k3s/server/manifests
+rm -rf "$MANIFESTS"
+mkdir -p "$MANIFESTS"
```

Comment:
Maybe add a comment that we only need to create this because our backend code assumes it exists? This isn't otherwise needed by k3s…